src/test/features/claim/routes/eligibility/not-eligible.ts

```diff
@@ -1,32 +0,0 @@
-import { expect } from 'chai'
-import * as request from 'supertest'
-import * as config from 'config'
-
-import { attachDefaultHooks } from '../../../../routes/hooks'
-import '../../../../routes/expectations'
-import { checkAuthorizationGuards } from '../checks/authorization-check'
-
-import { Paths as ClaimPaths } from 'claim/paths'
-
-import { app } from '../../../../../main/app'
-
-import * as idamServiceMock from '../../../../http-mocks/idam'
-
-const cookieName: string = config.get<string>('session.cookieName')
-
-describe('Claim eligibility: index page', () => {
```

Comment:
Why is this file removed? I would say it. should have got fixed not removed.

Reply:
The file did nothing and was just a copy paste error

Reply:
True, but it was supposed to cover `routes/eligibility/not-eligible.ts`. We should fix issues not remove tests.

Reply:
You're right, my mistake. adding now